dvc/repo/add.py

```diff
@@ -77,7 +100,12 @@ def _find_all_targets(repo, target, recursive):
 def _create_stages(repo, targets, fname, pbar=None):
     stages = []
 
-    for out in targets:
+    for out in Tqdm(
+        targets,
+        desc="Creating stages",
+        disable=True if len(targets) < LARGE_DIR_SIZE else None,
```

Comment:
Is LARGE_DIR_SIZE really needed here?

Reply:
might prevent unneeded flickering (creating and disappearing) of a really quick progressbar...